presto-main/src/main/java/com/facebook/presto/SessionRepresentation.java

```diff
@@ -45,6 +46,7 @@
     private final Optional<String> remoteUserAddress;
     private final Optional<String> userAgent;
     private final Optional<String> clientInfo;
+    private final Set<String> tags;
```

Comment:
we call this `clientTags` in `Session` we can use the same name here to be consistent.